### ISSUE TITLE
Error: Cannot read property 'credential' of undefined - with FacebookAuthProvider

### ISSUE THREAD

**Issue by jaimecorrea** [davecoates/rn-firebase-bridge#20]
My Code using Android, rn-firebase-bridge 0.0.5 and reac native 0.30.0

```
import { signInWithCredential } from 'rn-firebase-bridge/auth';
import { FacebookAuthProvider } from 'rn-firebase-bridge/auth';
...
AccessToken.getCurrentAccessToken().then(
                        (data) => {

                          let token = data.accessToken.toString();

                          let credential = FacebookAuthProvider.credential(token);

                          signInWithCredential(credential).then(
                            user => {
                              alert(user);
                            },
                            error => {
                              alert(error);
                            }
                          );
```

it seems FacebookAuthProvider is undefined but it was already imported from

**Comment by davecoates** [comment]
Hmm that looks right. What version of rn-firebase-bridge do you have installed?

**Comment by jaimecorrea** [comment]
Solved upgrading rn-firebase-bridge to 0.0.11...

Thanks!
